docs/self-managed/platform-deployment/helm-kubernetes/platforms/platforms.md

```diff
@@ -12,3 +12,7 @@ import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
 In addition to Stock Kubernetes (which could be deployed on cloud or on-premise), Camunda only officially tests and supports the following platforms:
 
 <DocCardList items={useCurrentSidebarCategory().items}/>
+
+:::caution Web Modeler
```

Comment:
I rephrased the warning. Please let me know what you think, @CatalinaMoisuc and @christian-konrad.

Reply:
Hm, I think I would keep my suggestion because it's a bit more concise and the meaning is basically the same. Hopefully, we don't need to display this warning for long anyway.